src/software/world/ball.cpp

```diff
@@ -15,7 +18,9 @@ Ball::Ball(const Point &position, const Vector &velocity, const Timestamp &times
 }
```

Comment:
Could we have this constructor delegate to the other constructor to avoid duplicating code?

Reply:
done